src/plugins/subsystems/drm/drm.sh

```diff
@@ -280,14 +282,27 @@ function get_available_connectors()
       ;;
   esac
 
-  while read -r card; do
-    card=$(basename "$card")
+  while read -r card_path; do
+    card=$(basename "$card_path")
     key=$(printf '%s\n' "$card" | grep card | cut -d- -f1)
     value=$(printf '%s\n' "$card" | grep card | cut -d- -f2)
     [[ "$key" == "$value" ]] && continue
 
     if [[ -n "$key" && -n "$value" ]]; then
       list_of_values="${cards[$key]}"
+      case "$target" in
+        2) # LOCAL TARGET
+          connector_enabled=$(< "$card_path"/enabled)
+          ;;
+        3) # REMOTE TARGET
+          get_enabled_cmd="cat $card_path/enabled"
+          connector_enabled=$(cmd_remotely "$get_enabled_cmd" "$flag")
```

Comment:
This approach will generate n SSH connections, which I think is not optimal and is also avoidable.
In the code before this part, we use the find command to identify all the connectors. We can leverage the `find` command to get the connector information and the enabled option in a single command, we can use something like this:

`find /sys/class/drm -name 'card*' -exec printf '%s - ' {} \; -exec cat {}/enabled \;`

P.S.: The above command is just a draft; you can probably improve it and maybe optimize it.